### ISSUE TITLE
Update `change-direct-deposit` content page

### ISSUE THREAD

**Issue by erikphansen** [department-of-veterans-affairs/va.gov-team#1296]
We need to change the content of the existing `change-direct-deposit` page (https://www.va.gov/change-direct-deposit/) from what it is now to the content of this Markdown file: https://github.com/department-of-veterans-affairs/vagov-content/blob/master/pages/change-direct-deposit-beta.md

## Background

The Direct Deposit info is being moved from eBenefits to the VA.gov profile page. When that change is live, we will need to update this supporting content page to reflect the change and point users to the new location of their Direct Deposit info.

## Timeline

The VSA Authenticated Experience team (PM @Justin-Pickett) will coordinate with the content team on when this content change will go live. But as of Monday, August 26, 2019 we are aiming for a mid-September release of the changes.

**Comment by peggygannon** [comment]
@Samara-Strauss @erikphansen @Justin-Pickett OK,thank you for the updates and clarifications! 

A few comments from me:

1. I will work with Kevin if I have any questions about adding a react widget to this page. I'm looking in the markdown file and I don't see a placeholder spot for the widget, but am assuming it will appear under the "Disability compensation and pension benefit payments" header. 

2. Have you determined the CTA widget language?
Are we assuming users are signed in and it should it say  "Go to my VA.gov profile" ?
Or is this widget asking user to sign in  "Sign in to VA.gov"?

3. I know we don't need this for beta, but do you have an estimate on the timing of the full launch?

**Comment by erikphansen** [comment]
@peggygannon 

1. The markdown file contains this bit of HTML: `<div class="cta-widget" data-widget-type="cta" data-app-id="direct-deposit"></div>` That's where the React Widget goes so you can treat that as a placeholder.
2. The CTA Widget language is hardcoded and the same across all instances of a Direct Deposit CTA widget
3. I'm not qualified to comment on this one!

**Comment by Samara-Strauss** [comment]
Hi @peggygannon!

> _I know we don't need this for beta, but do you have an estimate on the timing of the full launch?_

I believe mid-September. If UAT (happening next week) goes well and there aren't any significant bugs found, we can launch this quietly any time we want. However, we won't actively be pointing people to the feature until October, and we're coordinating with eBenefits to do that.

> _I will work with Kevin if I have any questions about adding a react widget to this page. I'm looking in the markdown file and I don't see a placeholder spot for the widget, but am assuming it will appear under the "Disability compensation and pension benefit payments" header._

The widget goes immediately under the "Disability compensation and pension benefit payments" heading like so:

![Direct deposit content page: Sign in prompt](https://user-images.githubusercontent.com/34068740/63944509-9ef33480-ca3f-11e9-95c2-31bf677260ca.png)

> _Have you determined the CTA widget language? Are we assuming users are signed in and it should it say "Go to my VA.gov profile" ? Or is this widget asking user to sign in "Sign in to VA.gov"?_

Yes — Danielle worked on the copy. Here is the copy for all four use cases:

## CTA to sign in

### Design

![Direct deposit content page: Sign in prompt](https://user-images.githubusercontent.com/34068740/58109247-4a782780-7bbb-11e9-9ccc-8a3cd4237f6d.png)

### Copy

**Please sign in to change your direct deposit information online**

Try signing in with your **DS Logon**, **My HealtheVet**, or **ID.me account**.  If you don’t have any of those accounts, you can create one now.

_[button]_ Sign in or create account

## Signed in, CTA to verify your identity

### Design

![Direct deposit content page: Verify your identity prompt](https://user-images.githubusercontent.com/34068740/58109267-5368f900-7bbb-11e9-831b-c76653f97c0e.png)

### Copy

**Please verify your identity to change your direct deposit information online**

We take your privacy seriously, and we’re committed to protecting your information. You’ll need to verify your identity before we can give you access to your bank account information. 

_[button]_ Verify your identity

## Signed in, identity verified, CTA to add MFA

### Design

![Direct deposit content page: Add 2FA prompt](https://user-images.githubusercontent.com/34068740/58109228-3f24fc00-7bbb-11e9-97fd-9490965e79a9.png)

### Copy

**Please set up 2-factor authentication to change your direct deposit information online**

We’re committed to protecting your information and preventing fraud. You’ll need to add an extra layer of security to your account with 2-factor authentication before we can give you access to your bank account information. 

_[button]_  Set up 2-factor authentication

## Signed in, identity verified, MFA added, CTA to go to profile

### Design

![Direct deposit content page: Go to profile](https://user-images.githubusercontent.com/34068740/58109281-57951680-7bbb-11e9-9428-ae0b0beb7615.png)

### Copy

**Go to your VA.gov profile to change your direct deposit information online**

Here, you can edit your bank name as well as your account number and type. 

_[button]_ Go to your profile

**Comment by peggygannon** [comment]
@Samara-Strauss and @Justin-Pickett I'm updating the direct deposit page today (adding widget and move down alert to lower on the page, etc)

@Samara-Strauss However, it's unclear to me how to account for the 4 different use cases above after the user signs in? Does the page widget change, based on the user being sign in vs. signed out and if they've been verified?

**Comment by Samara-Strauss** [comment]
@peggygannon please don't make any changes yet.

@lkoenigsberg can you confirm: we should **not** be changing the content page yet for DD since we are in a phased rollout but have not rolled out to all users.

> _However, it's unclear to me how to account for the 4 different use cases above after the user signs in? Does the page widget change, based on the user being sign in vs. signed out and if they've been verified?_

@peggygannon yes, the widget changes based on whether the user is signed out, signed in but LOA1, signed in/LOA3/no 2FA, or signed in/LOA3/has 2FA.

> _do we have to create 4 different page files for these scenarios?_

@erikphansen can you help with technical implementation when the time comes?

**Comment by Samara-Strauss** [comment]
@peggygannon @lkoenigsberg nvm, just saw the part of your post where you said you were keeping things in draft until full launch. Sorry I missed that at first.

But otherwise, all other comments from above still stand. I don't know anything about how the technical implementation should work, but @erikphansen may be able to help you there.

**Comment by erikphansen** [comment]
The widget is already done and will render all those states as needed. As long as it's embedded in the page it'll do everything itself.

**Comment by peggygannon** [comment]
@Samara-Strauss @erikphansen @Justin-Pickett OK, thank you for the updates. 

Yes, the page is in a "draft" state in the CMS and will NOT be published until full launch (please keep me posted on that date). 

And thank you for answering my questions about the "different" sign-in states. The widget is embedded in the draft page, so glad to hear it will render the different states. thank you!! 

@Samara-Strauss @Justin-Pickett do you want to see a PDF of the draft page?

**Comment by Samara-Strauss** [comment]
> do you want to see a PDF of the draft page?

Sure, why not 😄

**Comment by peggygannon** [comment]
![2019-09-27.png](https://images.zenhubusercontent.com/59b0414bb0222d5de476aa22/97f80f83-b0bb-4f68-8e35-f6d05335f43f)

Here's the screenshot, showing the updated direct-deposit widget.

**Comment by erikphansen** [comment]
probably. will need a new ticket for another change when we roll this out to 100% of users

**Comment by Samara-Strauss** [comment]
Great, I'll create a ticket.

**Comment by Samara-Strauss** [comment]
Created ticket to remove beta copy when we go live to 100% of users in #2511
